osu.Game/Rulesets/Objects/SliderPath.cs

```diff
@@ -31,7 +31,10 @@ public class SliderPath
         /// </summary>
         public readonly Bindable<double?> ExpectedDistance = new Bindable<double?>();
 
-        public bool HasValidLength => Precision.DefinitelyBigger(Distance, 0);
+        /// <summary>
+        /// Should be used to check whether placement can continue after a user editor operation.
+        /// </summary>
+        public bool HasValidLengthForPlacement => Precision.DefinitelyBigger(Distance, 0, 1);
```

Comment:
I've renamed this to better show that it's only used in the editor. I wanted to make these changes without affecting parsing logic to keep things simple, and this property felt a bit iffy with its previous name and location.